docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| 3  | Pressing a button is leading to opening a favourites_create a tag (12) page                                                                                                                                                                              |             |                                                                                    |
+| 4  | Image is the same as on model                                                                                                                                                                                                                            |             |                                                                                    |
+| 5  | Above text is a signature image of stars                                                                                                                                                                                                                 |             |                                                                                    |
+|    | 1.2 Favorite create a tag page 12                                                                                                                                                                                                                        |             |                                                                                    |
+| 6  | Text 1 line "create a tag:" 2 line "create a tag for adding translations to favorites in a well-organized way 3 line placeholder "e.g Lesson1"<br>4 line "You can always manage your tags, renaming, adding, and deleting them in the favorites section" |             |                                                                                    |
+| 7  |                                                                                                                                                                                                                                                          |             |
+| 8  | Texts are in the same format as in layout, text does not go out of screen                                                                                                                                                                                |             |                                                                                    |
+| 9  | Red button "create" is below all text                                                                                                                                                                                                                    |             |                                                                                    |
```

Comment:
Please correct the naming of buttons and other objects here and across the whole document according to the following rules:
1. All names of UI elements are in the quotes (double or single)
2. 'Create' button - first, the name of element in quotes and then the type of element. Other options are possible but let's make everything consistent. 
3. The first letters of the names of element should be capitalised.